tests/macros/fixtures/compat/alternative_name.json

```diff
@@ -0,0 +1,41 @@
+{
+    "webextensions": {
+        "api:": {
```

Comment:
This typo was difficult for me to recognize! Should be `"api"` not `"api:"`

Reply:
woah, sorry about that!